setup.py

```diff
@@ -18,16 +18,14 @@
 
 install_requires = [
     "google-auth-oauthlib >= 0.3.0, < 1.0.0",
-    # NOTE: Require google-api-core > 1.31.2 until
-    # https://github.com/googleapis/google-cloud-python/issues/10566
-    # is resolved. Once resolved, require google-api-core >=2.x.x
-    "google-api-core >= 1.31.2, < 3.0.0",
+    "google-api-core >= 2.0.1, < 3.0.0",
     "googleapis-common-protos >= 1.5.8, < 2.0.0",
     "grpcio >= 1.38.1, < 2.0.0",
     "proto-plus == 1.18.1",
     "PyYAML >= 5.1, < 6.0",
     "setuptools >= 40.3.0",
-    "nox >= 2020.12.31, < 2022.6",
+    "nox == 2020.12.31",
```

Comment:
@laurachevalier4 is this correct? I think I may want to revert this line, can you confirm?

Reply:
I think so, yes. We didn't want to pin a particular version but there was no "upper bound" that we could predict so we set an arbitrary future date when we might expect breaking changes to be introduced.

Reply:
I thought so, fixed this so it's the same as what's in the main branch now.